SPECS/host-installer.spec

```
@@ -1,16 +1,12 @@
 %global package_speccommit e2726e108c480a1796f01e3109580856df71189d
-%global package_srccommit v10.10.0
+%global package_srccommit v10.10.5.xcpng.1
 Summary: XenServer Installer
 Name: host-installer
-Version: 10.10.0
-Release: 1.1%{?xsrel}%{?dist}
+Version: 10.10.5.xcpng.1
+Release: 1.0%{?xsrel}%{?dist}
```

We don't need a zero.

```suggestion
Release: 1%{?xsrel}%{?dist}
```